### PR TITLE
Users should be able to update their profile details.

### PR DESCRIPTION
**Pull request by patrickngabo**
#### what does this PR do?
 - Update user profile
#### Description of the task to be completed
 - Users should be able to update their profile
 - write tests for testing user profile functionality
#### How should this be manually tested
 - To test this endpoint you need to use postman and type: ``/api/v1/edit-profile/:id``
 - Use the Patch method for sending a request
#### Any background context you want to provide
  - N/A
#### What are the relevant Trello story link
  -  [Users should be able to update their profile details.](https://trello.com/c/VCx3tNHS)
#### Screenshots (if appropriate)
Screenshot from 2021-01-26 11-53-16

#### Questions:
  - N/A

### CODE REVIEW

**Comment by fridolinho** [comment]
[Users should be able to update their profile details.](https://trello.com/c/VCx3tNHS/17-users-should-be-able-to-update-their-profile-details)

File: src/modules/account/__tests__/account.test.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import chaihttp from 'chai-http';
 import chai, { expect } from 'chai';
-import server from '../../server';
+import { _ } from 'lodash';
```
Comment (truestbyheart) [inline src/modules/account/__tests__/account.test.js:3]: Please just import what u need like `{ omit }` loading the whole library is too much

File: src/modules/account/account.controller.js [hunk 2]
```diff
@@ -40,6 +42,32 @@ class UserAccount {
...
+          data: editedProfile,
+        });
+      }
+      return res
+        .status(NOT_FOUND)
+        .json({ status: NOT_FOUND, message: 'User not found' });
+    } catch (error) {
+      return res.status(INTERNAL_SERVER_ERROR).json({
+        status: INTERNAL_SERVER_ERROR,
+        message: 'something went wrong',
```
Comment (truestbyheart) [inline src/modules/account/account.controller.js:67]: Please write a more descriptive error message, like `Fail to update profile`

**Review by truestbyheart — CHANGES_REQUESTED** [review]
Great work just a feddback or two and we are good to go

**Review by harerakalex — APPROVED** [review]
LGTM @patrickngabo

**Comment by luc-tuyishime** [comment]
Can you add a screenshot in the PR description ?

**Comment by patrickngabo** [comment]
> Can you add a screenshot in the PR description ?

Screenshot added

File: src/modules/account/__tests__/account.test.js (hunk 1 above)
Comment (patrickngabo) [reply src/modules/account/__tests__/account.test.js]: Import fixed.

File: src/modules/account/account.controller.js (hunk 2 above)
Comment (patrickngabo) [reply src/modules/account/account.controller.js]: error message fixed

**Pull request merged**